web/src/mobile/components/Orders/OrdersItem.tsx

```diff
@@ -41,29 +41,51 @@ const OrdersItemComponent = props => {
             </div>
             <div className="pg-mobile-orders-item__row">
                 <div className="pg-mobile-orders-item__row__block">
-                    <span>{intl.formatMessage({ id: 'page.mobile.orders.header.amount' })}</span>
-                    <span className="pg-mobile-orders-item__row__block__value">
-                        <Decimal fixed={currentMarket.amount_precision}>{order.origin_volume}</Decimal>
-                    </span>
+                    <span>{intl.formatMessage({ id: 'page.mobile.orders.header.filled' })}</span>
+                    <div className="pg-mobile-orders-item__row__block__value">
+                        <span style={{
+                            color: parseFloat(filled) < 100 ? 'var(--asks)' : 'var(--bids)'
+                        }}>
+                            {filled}%
```

Comment:
```suggestion
<Decimal fixed={FIXED_VOL_PRECISION}>{filled}</Decimal>%
```
import FIXED_VOL_PRECISION from constants.ts